### PR TITLE
add iconLake mainnet

### PR DESCRIPTION
**Pull request by huanxiangspace**

### CODE REVIEW

File: cosmos/iconlake-mainnet.json
Comment (editaahn) [inline cosmos/iconlake-mainnet.json:1]: The filename should be `iconlake.json`, following the rule for chain identifier and file name. You could get the details further from README.

Comment (huanxiangspace) [reply cosmos/iconlake-mainnet.json:1]: Thanks a lot. Just forgot.

**Comment by editaahn** [comment]
Thanks! Also, please upload the images to each image directory of the testnet and the mainnet.

**Comment by huanxiangspace** [comment]
> Thanks! Also, please upload the images to each image directory of the testnet and the mainnet.

I have added the images.

**Pull request merged**